CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.22.0](https://github.com/redhat-developer/quarkus-ls/milestone/26?closed=1) (April 17, 2025)
+
+### Enhancements
+- Warning instead of error (configurable) for unknown sections in Qute files. See [#1042](https://github.com/redhat-developer/quarkus-ls/issues/1042).
+- Support for Integer operators in Qute files. See [vscode-quarkus#974](https://github.com/redhat-developer/vscode-quarkus/issues/974).

```

Comment:
I don't think we've had quarkus-ls referencing vscode-quarkus issues. I would just reference the PR in quarkus-ls.

Reply:
Yep you're right, I switched it over